### PR TITLE
tests: Add standalone test for global options

### LINKED ISSUE #195: standalone tests for global options
include a standalone test for global options like version and help

### PR DESCRIPTION
**Pull request by GabyCT**
This will test global options like help and version

Fixes #195

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: functional/options_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+var _ = Describe ("global options", func() {
+	DescribeTable("option",
+		func(command *Command) {
+			exitCode := command.Run()
+
+			Expect(exitCode).To(Equal(0))
+			Expect(command.Stderr.String()).To(BeEmpty())
+			Expect(command.Stdout.String()).NotTo(BeEmpty())
+		},
+		withOption("--version"),
```
Comment (jodh-intel) [inline functional/options_test.go]: It would be good to sort this list, but that's a nit, so

lgtm.

File: functional/options_test.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		withOption("--v"),
+		withOption("--help"),
+		withOption("--h"),
```
Comment (jodh-intel) [inline functional/options_test.go]: Ah - actually, how about adding an invalid flag test (`--this-option-does-not-exist` or something) and ensure it fails?

**Comment by GabyCT** [comment]
@jodh-intel changes applied

**Comment by devimc** [comment]
lgtm

**Pull request merged**